frontend/src/components/Gallery/Gallery.jsx

```diff
@@ -24,14 +26,19 @@ const Gallery = ({ images }) => {
                   <div
                     className="image-wrapper"
                     key={`${img.id}-${idx}`}
-                    onMouseDown={() => handleShow(img)}
-                    role="button"
-                    tabIndex={idx}>
+                    tabIndex={idx}
+                  >
                     <img
                       className="search_gallery--image"
                       src={img.src}
                       alt={img.title}
                     />
+                    <button
+                      className="expand-button"
+                      onMouseDown={() => handleShow(img)}
+                    >
+                      <FontAwesomeIcon icon={faExpandAlt} inverse size="lg" />{" "}
```

Comment:
please remove `{" "}`, seems like we dont need this, or I am missing sth?

Reply:
No you're not missing a thing, I think prettier auto added that, I'll remove it. btw if you use prettier yourself and have a specific config, please let me know, if you don't I'll just disable it.

Reply:
@elyessAR  hmm I don't use it but you have a good point. This is a nice to have feature. Would you want to implement it for this project? If yes, please go ahead and create ticket. If not, please let me know so I can take a look at it. Sounds good?